modules/programs/broot.nix

```diff
@@ -138,6 +138,9 @@ let
         "broot --print-shell-function ${shell} > $out"
       }
     '';
+
+  defaultConfIsDirectory = builtins.compareVersions cfg.package.version "1.14.0"
+    >= 0;
```

Comment:
No need for this. We target Nixpkgs unstable and it doesn't have any older versions of broot packaged.

```suggestion
```